_includes/content/reqs-kernel.md

```diff
@@ -18,11 +18,13 @@
 - `xt_conntrack`
 - `xt_icmp` (for IPv4)
 - `xt_icmp6` (for IPv6)
-- `xt_ipvs`
+- `xt_ipvs`,`ipt_ipvs`
 - `xt_mark`
 - `xt_multiport`
 - `xt_rpfilter`
 - `xt_sctp`
 - `xt_set`
 - `xt_u32`
-- `ipip` (if using {{site.prodname}} networking)
\ No newline at end of file
+- `xt_bpf` (for eBPF)
+- `vfio-pci`
+- `ipip` (if using {{site.prodname}} networking)
```

Comment:
this should probably say "if using {{site.prodname}} networking in IPIP mode"

Reply:
I also added `wireguard` to the page.